django_s3_storage/storage.py

```diff
@@ -389,14 +389,21 @@ def listdir(self, path):
     def size(self, name):
         return self.meta(name)["ContentLength"]
 
+    def _presigned_url_params(self, name):
+        params = self._object_params(name)
+        content_disposition = _callable_setting(self.settings.AWS_S3_CONTENT_DISPOSITION, name)
+        if content_disposition:
+            params.update({ "ResponseContentDisposition", content_disposition })
```

Comment:
This is a `set()`, shouldn't this be a dictionary instead? (`{"ResponseContentDisposition": content_disposition}`)

Additionally, I think spaces aren't commonly used inside curly braces.